frontend/plugins/services/package.json

```diff
@@ -26,6 +26,12 @@
     "@material-ui/lab": "4.0.0-alpha.57",
     "@backstage/core": "^0.4.0",
     "@backstage/theme": "^0.2.2",
+    "@mdi/react": "1.4.0",
+    "@mdi/js": "5.8.55",
+    "numeral": "2.0.6",
+    "@types/numeral": "0.0.29",
+    "recharts": "^1.8.5",
+    "@types/recharts": "^1.8.18",
```

Comment:
Also, could you put `@types/numeral` and `@types/recharts` to devDependencies please?